src/Slider/README.md

```diff
@@ -115,3 +115,7 @@ const [selectedCount, totalCount] = calculateCountOf(histogramData, value, minVa
 
 console.log(`Showing ${selectedCount} of ${totalCount}`); // Showing 24 of 75 flights
 ```
+
+## Data-test
+
+There is a `dataTest` prop for ability to test the component, and there are also hardcoded `data-test` attributes on handlers in format `SliderHandle-${index}`.
```

Comment:
Can you specify the lowest index that will be every time rendered - when only one Handle? 🙏 @filipdanisko